docs/source/development/style-guide/requirement-and-issue-tracking.rst

```diff
@@ -0,0 +1,120 @@
+.. _requirement-and-issue-tracking:
+
+Requirement and Issue Tracking
+------------------------------
+
+Definitions
+^^^^^^^^^^^
+=================== ===================================================================================================================================================================================================================================================
+Name                Definition
+=================== ===================================================================================================================================================================================================================================================
+Level 4 requirement requirements that have been decomposed to the SDC level. Level 4 requirements have already been defined.
+Level 5 requirement requirements decomposed from the level 4 requirements that better define how the SDC plans to satisfy the parent level 4 requirement, while not defining any implementation details. Example: Backup *Metadata* to AWS backup account vs. Backup *OpenSearch* to AWS backup account
+Implementation task Github issue that better defines how a certain feature will be implemented. Implementation tasks are tracked under the parent level 5 requirement as a sub-task. Each implementation task should be small enough to be handled in a single PR.
+SPIKE               Github issue created to investigate different architecture design approaches. Like implementation tasks, SPIKEs should be tracked under the applicable level 5 requirement as a sub-task.
+=================== ===================================================================================================================================================================================================================================================
+
+Requirements Overview
+^^^^^^^^^^^^^^^^^^^^^
+
+Requriements Mapping
+~~~~~~~~~~~~~~~~~~~~
+
+* Each level 4 requirement is mapped to at least one SIT.
+* Each level 4 requirement relevant to the upcoming SIT is decomposed into level 5 requirements that, together, cover the testing requirements of the SIT.
+* The level 5 requirement creation is incremental as we move through development for each SIT.
+* Each Level 5 requirement will have implementation tasks (sub-issues) that fully cover the Level 5 requirement and contain implementation specific information.
+
+.. image:: ../../_static/req-decomposition.png
+   :alt: Requirement decomposition diagram
+   :width: 1293
+   :height: 629
+
+Requirements Lifecycle
+~~~~~~~~~~~~~~~~~~~~~~
+The below image shows an example of the lifecycle of a Level 4 requirement over the course of the project:
+
+.. image:: ../../_static/req-lifecycle.png
+   :alt: Requirement lifecycle diagram
```

Comment:
This diagram is quite squished on my screen. Maybe because I zoomed in to make the text bigger? Not sure how to fix it, but maybe you can look into a more responsive size option (like making it take the full width of the screen, or something)

Reply:
Yeah, I guess if there's a way to have the left index and right table of contents closer to the end of the window that would create more space for the images. I can look into if there's a way to do that.